tests/test_scala.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+
+@pytest.fixture()
+def AnsibleDefaults(Ansible):
+    return Ansible("include_vars", "./defaults/main.yml")["ansible_facts"]
+
+
+@pytest.fixture()
+def AnsibleFacts(Ansible):
+    return Ansible("setup")["ansible_facts"]
+
+
+def test_java_exists(AnsibleFacts, Package):
+
+    # Ubuntu 16.04 and above only support openjdk 8
+    os_version = AnsibleFacts["ansible_distribution_version"]
+    if float(os_version) >= 16.04:
+        java_major_version = "8"
+    else:
+        java_major_version = "7"
+
+    assert Package("openjdk-%s-jdk" % java_major_version).is_installed
+
+
+def test_scala_exists(Command):
+    Command.run_test(
+        "scala -e \"println(util.Properties.versionNumberString)\"")
+
+
+def test_scala_version(Command, AnsibleDefaults):
+    # Extract target scala version from ansible facts
```

Comment:
I think these comments are fine as is, but wanted to point out some of the loose `pydoc` documentation standards from Google we try to follow.

See: https://google.github.io/styleguide/pyguide.html#Comments